### PR TITLE
docs(llm): proofread

### PR DESCRIPTION
**Pull request by tgenaitay**
### Your checklist for this pull request

- [X] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Specifies some features.
Brings new details on models.
Adds links.

### CODE REVIEW

File: ai-data/llm-inference/how-to/create-deployment.mdx [hunk 1]
```diff
@@ -15,20 +15,21 @@ dates:
...
           Some models may require acceptance of an end-user license agreement. If prompted, review the terms and conditions and accept the license accordingly.
         </Message>
     - Choose the geographical **region** for the deployment.
     - Specify the GPU Instance type to be used with your deployment.
-4. Enter a **name** for the deployment and, optionally, description and tags.
+4. Enter a **name** for the deployment and, optionally, tags.
 5. Configure the **network** settings for the deployment:
     - Enable **Private Network** for secure communication and restricted availability within Private Networks. Choose an existing Private Network from the drop-down list, if applicable.
-    - Enable **Public Network** to access resources via the public IP address of the GPU Instance.
+    - Enable **Public Network** to access resources via the public Internet. A token protection is enabled by default.
```
Comment (RoRoJ) [inline ai-data/llm-inference/how-to/create-deployment.mdx:29]:
```suggestion
    - Enable **Public Network** to access resources via the public Internet. Token protection is enabled by default.
```

File: ai-data/llm-inference/quickstart.mdx [hunk 2]
```diff
@@ -10,16 +10,21 @@ categories:
   - ia
 ---
 
-Scaleway LLM Inference service, a scalable and secure inference engine for machine learning models.
+Scaleway LLM Inference service, the 1st European LLM Inference platform on the market, is a scalable and secure inference engine for large language models.
```
Comment (RoRoJ) [inline ai-data/llm-inference/quickstart.mdx:13]:
```suggestion
Scaleway LLM Inference service, the first European LLM Inference platform on the market, is a scalable and secure inference engine for large language models.
```

File: ai-data/llm-inference/quickstart.mdx [hunk 3]
```diff
@@ -10,16 +10,21 @@ categories:
...
 
-Here are some of the key features of Scaleway Inference:
+Here are some of the key features of Scaleway Inference service:
 
-* **Easy deployment**: Deploy your inference models with just a few clicks. Scaleway Inference provides a simple and intuitive interface for uploading and deploying models.
-* **Auto-scaling**: Scaleway LLM Inference automatically scales your inference instances based on the demand, ensuring that your models are always available and responsive.
-* **Security**: Scaleway provides a secure environment for running your inference models. Our platform is built on top of a secure architecture, and we use state-of-the-art security.
+* **Easy deployment**: Deploy state-of-the-art open weights LLMs with just a few clicks. Scaleway Inference provides a simple and intuitive interface for generating dedicated endpoints.
+* **Security**: Scaleway provides a secure environment for running your models. Our platform is built on top of a secure architecture, and we use state-of-the-art security.
+* **Complete data privacy**: No storage nor third-party access to your data (prompt & responses), ensuring it remains exclusively yours and within Europe.
```
Comment (RoRoJ) [inline ai-data/llm-inference/quickstart.mdx:22]:
```suggestion
* **Complete data privacy**: No storage or third-party access to your data (prompt or responses), to ensure it remains exclusively yours and stays within Europe.
```

File: ai-data/llm-inference/quickstart.mdx [hunk 4]
```diff
@@ -10,16 +10,21 @@ categories:
...
+* **Auto-scaling (soon!)**: Scaleway LLM Inference automatically scales your instances based on the demand, ensuring that your models are always available and responsive.
```
Comment (RoRoJ) [inline ai-data/llm-inference/quickstart.mdx:23]:
```suggestion
* **Auto-scaling (coming soon)**: Scaleway LLM Inference automatically scales your instances based on demand, ensuring that your models are always available and responsive.
```

File: ai-data/llm-inference/quickstart.mdx [hunk 5]
```diff
@@ -10,16 +10,21 @@ categories:
...
+
+<Message type="important">
+  This service is in private beta. [Specific Terms and Conditions](https://www.scaleway.com/en/contracts/) are applicable.
```
Comment (RoRoJ) [inline ai-data/llm-inference/quickstart.mdx:26]:
```suggestion
  This service is in private beta. [Terms and conditions](https://www.scaleway.com/en/contracts/) apply.
```

File: ai-data/llm-inference/quickstart.mdx [hunk 6]
```diff
@@ -28,21 +33,22 @@ Here are some of the key features of Scaleway Inference:
...
           Some models may require acceptance of an end-user license agreement. If prompted, review the terms and conditions and accept the license accordingly.
         </Message>
     - Choose the geographical **region** for the deployment.
     - Specify the GPU Instance type to be used with your deployment.
-4. Enter a **name** for the deployment, along with an optional description and tags to aid in organization.
+4. Enter a **name** for the deployment, along with optional tags to aid in organization.
 5. Configure the **network** settings for the deployment:
     - Enable **Private Network** for secure communication and restricted availability within Private Networks. Choose an existing Private Network from the drop-down list, if applicable.
-    - Enable **Public Network** to access resources via the public IP address of the GPU Instance.
+    - Enable **Public Network** to access resources via the public Internet. A token protection is enabled by default.
```
Comment (RoRoJ) [inline ai-data/llm-inference/quickstart.mdx:47]:
```suggestion
    - Enable **Public Network** to access resources via the public Internet. Token protection is enabled by default.
```

File: ai-data/llm-inference/quickstart.mdx [hunk 7]
```diff
@@ -72,5 +78,5 @@ Here are some of the key features of Scaleway Inference:
 5. Type **DELETE** to confirm and click **Delete deployment** to delete your deployment. 
  
 <Message type="important">
-  Deleting a deployment is a permanent action and erases all its associated data.
+  Deleting a deployment is a permanent action, erasing all its associated configuration and resources.
```
Comment (RoRoJ) [inline ai-data/llm-inference/quickstart.mdx:81]:
```suggestion
  Deleting a deployment is a permanent action, and erases all its associated configuration and resources.
```

File: ai-data/llm-inference/reference-content/llama-2-70b.mdx [hunk 8]
```diff
@@ -70,6 +64,9 @@ curl -k --request POST \
 
 ### Receiving Inference responses
 
-Upon sending the HTTP request to the public or private endpoints exposed by the server, you will receive inference responses from the Triton server. 
+Upon sending the HTTP request to the public or private endpoints exposed by the server, you will receive inference responses from the managed Inference server. 
 Process the output data according to your application's needs. The response will contain the output generated by the LLM model based on the input provided in the request.
 
+<Message type="note">
+  Despite efforts for accuracy, the possibility of generated text containing inaccuracies or 'hallucinations' exists. Always independently verify the content generated.
```
Comment (RoRoJ) [inline ai-data/llm-inference/reference-content/llama-2-70b.mdx:71]:
```suggestion
  Despite efforts to ensure accuracy, the possibility of generated text containing inaccuracies or 'hallucinations' exists. Always independently verify the content generated.
```

File: ai-data/llm-inference/reference-content/wizardlm-70b.mdx [hunk 9]
```diff
@@ -65,5 +61,9 @@ curl -k --request POST \
...
 ### Receiving Inference responses
 
-Upon sending the HTTP request to the public or private endpoints exposed by the server, you will receive inference responses from the Triton server. 
-Process the output data according to your application's needs. The response will contain the output generated by the LLM model based on the input provided in the request.
\ No newline at end of file
+Upon sending the HTTP request to the public or private endpoints exposed by the server, you will receive inference responses from the managed Inference server. 
+Process the output data according to your application's needs. The response will contain the output generated by the LLM model based on the input provided in the request.
+
+<Message type="note">
+  Despite efforts for accuracy, the possibility of generated text containing inaccuracies or 'hallucinations' exists. Always independently verify the content generated.
```
Comment (RoRoJ) [inline ai-data/llm-inference/reference-content/wizardlm-70b.mdx:68]:
```suggestion
  Despite efforts to ensure accuracy, the possibility of generated text containing inaccuracies or 'hallucinations' exists. Always independently verify the content generated.
```

File: ai-data/llm-inference/quickstart.mdx (hunk 4 above)
Comment (jcirinosclwy) [reply ai-data/llm-inference/quickstart.mdx:23]: Instances here should be capitalized? @RoRoJ

**Review by jcirinosclwy — APPROVED** [review]

File: ai-data/llm-inference/quickstart.mdx (hunk 5 above)
Comment (tgenaitay) [reply ai-data/llm-inference/quickstart.mdx:26]: It was meant to say we have specific terms as opposed to the general terms of services. In fact, the beta specific terms take precedence over our general terms of service.

Comment (RoRoJ) [reply ai-data/llm-inference/quickstart.mdx:26]: OK I've added back the word specific

File: ai-data/llm-inference/how-to/create-deployment.mdx [hunk 10]
```diff
@@ -15,20 +15,21 @@ dates:
...
+        - [Llama-2-70b](/ai-data/llm-inference/reference-content/llama-2-70b)
+        - [Mixtral-8x7B-Instruct-v0.1](/ai-data/llm-inference/reference-content/mixtral-8x7b)
+        - [WizardLM-70b-V1.0](/ai-data/llm-inference/reference-content/wizardlm-70b)
         <Message type="note">
           Some models may require acceptance of an end-user license agreement. If prompted, review the terms and conditions and accept the license accordingly.
         </Message>
     - Choose the geographical **region** for the deployment.
     - Specify the GPU Instance type to be used with your deployment.
-4. Enter a **name** for the deployment and, optionally, description and tags.
+4. Enter a **name** for the deployment and, optionally, tags.
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/create-deployment.mdx:26]:
```suggestion
4. Enter a **name** for the deployment, and optional tags.
```

File: ai-data/llm-inference/how-to/create-deployment.mdx [hunk 11]
```diff
@@ -15,20 +15,21 @@ dates:
...
 5. Configure the **network** settings for the deployment:
     - Enable **Private Network** for secure communication and restricted availability within Private Networks. Choose an existing Private Network from the drop-down list, if applicable.
-    - Enable **Public Network** to access resources via the public IP address of the GPU Instance.
+    - Enable **Public Network** to access resources via the public Internet. Token protection is enabled by default.
```
Comment (nerda-codes) [inline ai-data/llm-inference/how-to/create-deployment.mdx:29]:
```suggestion
    - Enable **Public Network** to access resources via the public internet. Token protection is enabled by default.
```

File: ai-data/llm-inference/quickstart.mdx [hunk 12]
```diff
@@ -28,21 +33,22 @@ Here are some of the key features of Scaleway Inference:
 2. Click on **Create deployment** to launch the deployment creation wizard.
```
Comment (nerda-codes) [inline ai-data/llm-inference/quickstart.mdx:33]:
```suggestion
2. Click **Create deployment** to launch the deployment creation wizard.
```

File: ai-data/llm-inference/reference-content/llama-2-70b.mdx [hunk 13]
```diff
@@ -44,24 +45,17 @@ Its robust performance and natural language understanding enhance user experienc
...
-<Message type="tip">
-  Ensure you have the necessary libraries provided by NVIDIA for Triton to interact with the server from your client code.
-  You can find these libraries on [GitHub](https://github.com/triton-inference-server/client/tree/main).
-  Follow the instructions provided in the `README` file to integrate these libraries into your project.
-</Message>
-
 ### Sending Inference requests
 
-To perform inference tasks with your LLM model deployed on the Triton Inference Server, follow these steps:
+To perform inference tasks with your LLM model deployed at Scaleway, follow these steps:
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/llama-2-70b.mdx:50]:
```suggestion
To perform inference tasks with your LLM model deployed at Scaleway, use the following command:
```

File: ai-data/llm-inference/reference-content/mixtral-8x7b.mdx [hunk 14]
```diff
@@ -40,24 +42,17 @@ Whether you are developing virtual assistants or educational platforms, Mixtral-
...
-<Message type="tip">
-  Ensure you have the necessary libraries provided by NVIDIA for Triton to interact with the server from your client code.
-  You can find these libraries on [GitHub](https://github.com/triton-inference-server/client/tree/main).
-  Follow the instructions provided in the `README` file to integrate these libraries into your project.
-</Message>
-
 ### Sending Inference requests
 
-To perform inference tasks with your LLM model deployed on the Triton Inference Server, follow these steps:
+To perform inference tasks with your LLM model deployed at Scaleway, follow these steps:
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/mixtral-8x7b.mdx:47]:
```suggestion
To perform inference tasks with your LLM model deployed at Scaleway, use the following command:
```

File: ai-data/llm-inference/reference-content/wizardlm-70b.mdx [hunk 15]
```diff
@@ -40,23 +42,17 @@ Its adaptability and natural language fluency make it valuable for enhancing pro
...
-<Message type="tip">
-  Ensure you have the necessary libraries provided by NVIDIA for Triton to interact with the server from your client code.
-  You can find these libraries on [GitHub](https://github.com/triton-inference-server/client/tree/main).
-  Follow the instructions provided in the `README` file to integrate these libraries into your project.
-</Message>
-
 ### Sending Inference requests
 
-To perform inference tasks with your LLM model deployed on the Triton Inference Server, follow these steps:
+To perform inference tasks with your LLM model deployed at Scaleway, follow these steps:
```
Comment (nerda-codes) [inline ai-data/llm-inference/reference-content/wizardlm-70b.mdx:47]:
```suggestion
To perform inference tasks with your LLM model deployed at Scaleway, use the following command:
```

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**